src/cobertura.rs

```diff
@@ -786,7 +792,7 @@ mod tests {
 
         let results = read_file(&file_path);
 
-        println!("{}\n", results);
```

Comment:
Nit: please keep the print, it can be useful for debugging

Reply:
This line actually slipped in accidentally with commit 8048843df2ebc08c0f56d7e12da56d8519f25b94. No other tests output the resulting XML to stdout.

Of course if you want me to keep it, I can do that. But then I'd prefer to add it for all tests.